aspnetcore/tutorials/razor-pages/new-field.md

```diff
@@ -45,13 +45,13 @@ Update the following pages:
 * Update [Create.cshtml](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Pages/Movies/Create.cshtml) with a `Rating` field.
 * Add the `Rating` field to the Edit Page.
 
-The app won't work until the DB is updated to include the new field. If run now, the app throws a `SqlException`:
+The app won't work until the DB is updated to include the new field. Running the app now would throw a `SqlException`:
 
 `SqlException: Invalid column name 'Rating'.`
 
-This error is caused by the updated Movie model class being different than the schema of the Movie table of the database. (There's no `Rating` column in the database table.)
+This exception is caused by the updated Movie model class being different than the schema of the Movie table of the database. (There's no `Rating` column in the database table.)
 
-There are a few approaches to resolving the error:
+There are a few approaches to preventing this exception:
```

Comment:
You're really not preventing an exception. It's really an error in what's been done that throws an exception. It's assuming the exception has occurred.